hw_diag/utilities/variant_definitions.py

```diff
@@ -0,0 +1,215 @@
+variant_definitions = {
+    # Nebra Indoor Hotspot Gen1
+    'NEBHNT-IN1': {
+        'FRIENDLY': 'Nebra Indoor Hotspot Gen 1',
+        'APPNAME': 'Indoor',
+        'SPIBUS': 'spidev1.2',
+        'RESET': 38,
+        'MAC': 'eth0',
+        'STATUS': 25,
+        'BUTTON': 26,
+        'ECCOB': True,
+        'TYPE': 'Full'
+        },
+
+    # Nebra Indoor Hotspot, Old identifier
+    'Indoor': {
+        'FRIENDLY': 'Nebra Indoor Hotspot Gen 1',
+        'APPNAME': 'Indoor',
+        'SPIBUS': 'spidev1.2',
+        'RESET': 38,
+        'MAC': 'eth0',
+        'STATUS': 25,
+        'BUTTON': 26,
+        'ECCOB': True,
+        'TYPE': 'Full'
+        },
+
+    # Nebra Outdoor Hotspot Gen1
+    'NEBHNT-OUT1': {
+        'FRIENDLY': 'Nebra Outdoor Hotspot Gen 1',
+        'APPNAME': 'Outdoor',
+        'SPIBUS': 'spidev1.2',
+        'RESET': 38,
+        'MAC': 'eth0',
+        'STATUS': 25,
+        'BUTTON': 26,
```

Comment:
BUTTON needs changing to 24

See https://github.com/NebraLtd/hm-pktfwd/pull/31

Reply:
Now packaged Enjoy https://pypi.org/project/hm-hardware-defs/

You want to use v0.1.4

See also https://github.com/NebraLtd/hm-pktfwd/pull/32